web-animations-1/Overview.bs

```diff
@@ -629,13 +637,11 @@ Timing model characteristics {#timing-model-overview}
 Time values {#time-value-section}
 -----------
 
-  Timing is based on a hierarchy of time relationships between timing nodes.
-  Parent nodes provide timing information to their child nodes
-  in the form of [=time values=].
-
-  A <dfn>time value</dfn> is a real number which nominally represents
+  A <dfn>time value</dfn> in the [=Web Animations model=]
+  is a value that serves the purpose of synchronization.
```

Comment:
"serves the purpose of synchronization" ?

I'm not sure what this is saying. Would you care to elaborate?

Reply:
It's a bit weird, I agree; I pulled this definition from below, see 

> ```
> A <dfn export>timeline</dfn> provides a source of [=time values=]
>  for the purpose of synchronization.
> ```

The problem I'm trying to solve here is giving “time value” a definition that works equally well in WA2 where we have non-time-based “time values” ...

Reply:
Yeah, I can see that. It's tricky. It's almost like the animations equivalent of a length. Maybe what you have is fine.

Reply:
Ok, we'll roll with this for now, and if anyone comes up with something better we can swap it in. :)